00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.02 Design Decisions/hub.yaml proposal for themes.md

```diff
@@ -0,0 +1,303 @@
+        callouts:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        checklists:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        images: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        tables: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        asides: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        highlights: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        color_schemes: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        snippets: [] # Path to css snippets relative to the root of the repo. We could parse the css similar to style settings
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings, e.g. pointing to the css files for download
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        css_classes: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        miscellaneous: [] # Other features you want to mention as markdown-formatted strings
+    plugin_support: [] # IDs of the plugins that have been styled by this theme (adding them here would ignore the plugin IDs on the obsidian.css file)
```

Comment:
this one I am not too sure what do to about. Problem is that plugin compatibility is only relevant for the 10k club, since they change so much. A simple theme only changing some colors will always be compatible to every plugin, since it's barely different from the default theme.

I think there should be some way to indicate this type of information? 🤔

Reply:
Maybe it's about phrasing... The plugins listed here should be those that have been tested with/purposefully styled, because yeah, _technically_ all themes are compatible with all plugins.

Reply:
What do we do when the list is present in both CSS and the yaml? I'd prefer it to be only in one place - and if they can list it in the YAML, then that would be the one to retain.